src/App.vue

```diff
@@ -424,6 +426,8 @@ export default {
     <v-main :class="{ 'pt-0': isWelcome }">
       <v-progress-linear absolute :active="loading" indeterminate height="5" />
 
+      <WorkQueueBanner />
```

Comment:
For me this appears in the same spot at the current alert.  Could we organize them?

Reply:
I think they should organize themselves by default, are they overlapping?

Reply:
Ah nope, didn't realize the material spec dictates that shouldn't be the case. I'll move this one so they shouldn't overlap with our normal ones 👍🏻

Reply:
![Screen Shot 2021-05-17 at 7 40 32 PM](https://user-images.githubusercontent.com/27291717/118569355-cb853d80-b747-11eb-84c9-aa9c759b5127.png)

Errors and notifications using the existing `Alert` component should now be properly nudged up.